libcxx/test/std/experimental/simd/simd.class/simd_ctor_conversion.pass.cpp

```diff
@@ -8,6 +8,7 @@
 
 // UNSUPPORTED: c++03, c++11, c++14
 // XFAIL: target=powerpc{{.*}}le-unknown-linux-gnu
+// ADDITIONAL_COMPILE_FLAGS: -O3
```

Comment:
We should remove this before landing this patch. We don't want to hardcode the optimization setting in this test. When I land my PR we'll have a CI configuration that tests everything with optimizations enabled so it'll cover that.